### PR TITLE
mongo db connected

### PR DESCRIPTION
**Pull request by lightning0125**
mongo db connected

### CODE REVIEW

**Comment by lightning0125** [comment]
I added mongodb connect function

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@ node_modules
 /dist
 
 # local env files
+.env
```
Comment (k-star-229) [inline .gitignore:6]: Looks great! @lightning0125

File: variables.env [hunk 2]
```diff
@@ -0,0 +1 @@
+MONGO_URI=mongodb+srv://root:root@cluster0.ds5ft7h.mongodb.net/?retryWrites=true&w=majority
```
Comment (k-star-229) [inline variables.env:1]: remove this file

**Review by k-star-229 — APPROVED** [review]
can approve, but remove .env file

File: .gitignore (hunk 1 above)
Comment (k-star-229) [reply .gitignore:6]: Thanks

File: variables.env (hunk 2 above)
Comment (k-star-229) [reply variables.env:1]: Oh, I am sorry

**Pull request merged**